pages/projects/index.vue

```diff
@@ -29,14 +29,16 @@
               </v-card-text>
               <v-card-actions>
                 <v-row justify="center">
-                  <v-btn
-                    :disabled="isDisabled"
-                    color="primary"
-                    rounded
-                    @click="createProject"
-                  >
-                    Create Project
-                  </v-btn>
+                  <nuxt-link :to="`projects/${newProject.id.toString()}`">
```

Comment:
I dont think it's a good idea because i still will be able to click a link even if button is disabled